src/components/ListBookings/ListBookings.jsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable linebreak-style */
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { FetchedBookings } from '../../redux/bookings/bookings';
+import CardBooking from '../CardBooking/CardBooking';
+import './ListBookings.css'
+const ListBookings = () => {
+  const dispatch = useDispatch();
+  const bookings = useSelector((state) => state.bookings.data);
+  const status = useSelector((state) => state.bookings.status);
+  useEffect(() => {
+    dispatch(FetchedBookings());
+  }, [dispatch]);
+  console.log(bookings);
+  console.log(status);
```

Comment:
- [ ] Kindly remove this old piece of debugging console.log statement, in order to improve the repo's cleanliness. 👍